### PR TITLE
[4408] Update channel list docs

### PR DESCRIPTION
**Pull request by bychkovdmitry**
### 🎯 Goal

Closes https://github.com/GetStream/stream-chat-android/issues/4408

Update channel list docs.

### 🎨 UI Changes

screencapture-localhost-3000-chat-docs-sdk-android-ui-channel-components-channel-list-2022-11-29-13_19_32

### 🧪 Testing

Launch docusaurus with `npx stream-chat-docusaurus -i -s`, and ensure that the changes look good.

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [x] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### CODE REVIEW

**Review by MarinTolic — CHANGES_REQUESTED** [review]
Requesting changes due to the agreement that we should not cut the channel list from the screens but rather frame it.

Also, I couldn't get the shimmer/ loading state tutorial to work in a standalone app, but might be a mistake on my end.

File: docusaurus/docs/Android/03-ui/03-channel-components/02-channel-list.mdx [hunk 1]
```diff
@@ -157,8 +157,9 @@ The full list of available XML attributes is available under `ChannelListView` s
 The following example shows how to modify the style of all `ChannelListView` instances globally to:
 
 * Disable the default options
+* Change the foreground color
+* Change the read indicator icon
```
[the remark this answers is not in the document]
Comment (bychkovdmitry) [reply docusaurus/docs/Android/03-ui/03-channel-components/02-channel-list.mdx:161]: I'll leave it as is, as mutating a drawable is too complex for this example

File: docusaurus/docs/Android/03-ui/03-channel-components/02-channel-list.mdx [hunk 2]
```diff
@@ -3,33 +3,33 @@ import TabItem from '@theme/TabItem';
...
-<io.getstream.chat.android.ui.channel.list.ChannelListView
-    android:id="@+id/channelsView"
+<io.getstream.chat.android.ui.feature.channels.list.ChannelListView
+    android:id="@+id/channelListView"
     android:layout_width="match_parent"
     android:layout_height="match_parent" />
 ```
 
-We recommend using the `ChannelListViewModel` to get the full list of data from the Stream API and then render it using the `ChannelListView`.
+We recommend fetching data from Stream's API using `ChanneListViewModel`, and then rendering it inside `ChannelListView.`
```
Comment (MarinTolic) [inline docusaurus/docs/Android/03-ui/03-channel-components/02-channel-list.mdx:32]:
```suggestion
We recommend fetching data from Stream's API using `ChanneListViewModel`, and then rendering it inside `ChannelListView`.
```

File: docusaurus/docs/Android/03-ui/03-channel-components/02-channel-list.mdx [hunk 3]
```diff
@@ -80,7 +80,7 @@ ChannelListViewModelBinding.bind(viewModel, channelListView, getViewLifecycleOwn
 You may need to pass a custom `ChatEventHandler` to make sure the list is updated properly. Check [ChannelListUpdates](../../02-client/06-guides/05-channel-list-updates.mdx) section to read more.
 :::
 
-All the logic of subscribing to data emitted by the ViewModel is provided by the `bindView` function. Other than channel data loading, the ViewModel is also handling actions like deleting a channel and leaving a group conversation by default.
+All the logic of subscribing to data emitted by the ViewModel is provided by the bindView function. Other than channel data loading, the ViewModel also handles actions like deleting a channel and leaving a group conversation by default.
```
Comment (MarinTolic) [inline docusaurus/docs/Android/03-ui/03-channel-components/02-channel-list.mdx:83]:
```suggestion
All the logic of subscribing to data emitted by the ViewModel is provided by the `bindView` function. Other than channel data loading, the ViewModel also handles actions like deleting a channel and leaving a group conversation by default.
```

**Review by MarinTolic — APPROVED** [review]

**Pull request merged**